docs/workshop/content/cloning.md

```diff
@@ -529,32 +515,32 @@ spec:
       volumes:
         - name: disk0
           persistentVolumeClaim:
-            claimName: fc31-clone
+            claimName: fc34-clone
 EOF
 
-virtualmachine.kubevirt.io/fc31-clone created
+virtualmachine.kubevirt.io/fc34-clone created
 ~~~
 
 After a few minutes you should see the new virtual machine running:
 
 ~~~bash
 $ oc get vmi
 NAME                    AGE     PHASE       IP                  NODENAME
-fc31-clone              6m15s   Running     192.168.123.67/24   ocp4-worker2.cnv.example.com
-fc31-nfs                37m     Succeeded   192.168.123.64/24   ocp4-worker1.cnv.example.com
-rhel8-server-hostpath   109m    Running     192.168.123.63/24   ocp4-worker1.cnv.example.com
-rhel8-server-nfs        111m    Running     192.168.123.62/24   ocp4-worker1.cnv.example.com
+NAME               AGE    PHASE       IP                NODENAME
+fc34-clone         59s    Running     192.168.123.61    ocp4-worker2.cnv.example.com
+fc34-nfs           50m    Succeeded   192.168.123.60    ocp4-worker2.cnv.example.com
+rhel8-server-nfs   153m   Running     192.168.178.117   ocp4-worker2.cnv.example.com
 ~~~
 
 > **Note** Give the command 2-3 minutes to report the IP. 
 
 This machine will also be visible from the console, and you can login using "**root/redhat**" just like before:
 
-<img src="img/fc31-clone-console.png"/>
+<img src="img/fc34-clone-console.png"/>
```

Comment:
Does this file exist?

Reply:
I believe it is in the seperate commit caf5303